jekyll/_includes/head.html

```diff
@@ -21,17 +21,14 @@
 	<link rel="stylesheet" href="{{ "/assets/css/footer.css" | prepend: site.baseurl }}" type="text/css" media="screen" />
 	<link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css" />
 	<link rel="stylesheet" href="{{ "/assets/css/asciidoc.css" | prepend: site.baseurl }}">
-	<!--link rel="stylesheet" href="{{ "/assets/css/asciidoc-pygments.css" | prepend: site.baseurl }}"-->
-	<!--link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/default.min.css">
 	<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/highlight.min.js"></script>
-	<script>hljs.initHighlightingOnLoad();</script-->
+	<script>hljs.initHighlightingOnLoad();</script>
```

Comment:
Could we have this somewhere else? like somewhere in `main.js` maybe?

Reply:
Done!